build/dsc.rb

```diff
@@ -4,11 +4,15 @@
 require 'pathname'
 require 'erb'
 require 'open-uri'
-require 'mof'
-require 'cim'
 require 'find'
 require 'logger'
-require 'charlock_holmes/string'
+
+# Bundler group 'build'
+if Bundler.rubygems.find_name('mof').any?
```

Comment:
This is actually more of a 'what gems are installed' problem, than a 'Windows problem'.

It just so happens that Windows can't install the `build` gem group... but if you did a `bundle install --path .bundle/gems --without build` on any other OS, you'd have the same problem running rake tasks, which is why I opted to guard it this way.